data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/grok.mapping.yml

```diff
@@ -0,0 +1,11 @@
+pluginName: grok
```

Comment:
This looks good. If any changes are made to the naming before release I will modify this file or let you know what the changes are.

Reply:
What happens if there is something that we don't support (such as `tag_on_timeout`. Do name mappings only need to be made for configuration settings that exist in Data Prepper?